setup.py

```diff
@@ -55,6 +55,13 @@
         "Programming Language :: Python :: Implementation :: CPython",
         "Programming Language :: Python :: Implementation :: PyPy",
     ],
+    
+    project_urls={
+        'Homepage': 'https://signac.io'
```

Comment:
```suggestion
        'Homepage': 'https://signac.io',
```
This is why the CI pre-checks are failing.

Reply:
My mistake, I was fixing other pre-checks. 😆